sdk/aqueduct/backend/api_client.py

```diff
@@ -339,8 +341,11 @@ def register_airflow_workflow(
     def _construct_register_workflow_request(
         self,
         dag: DAG,
+        run_now: bool,
     ) -> Tuple[Dict[str, str], Dict[str, str], Dict[str, IO[Any]]]:
         headers = self._generate_auth_headers()
+        # This header value will be string "True" or "False"
+        headers.update({"run-now": str(run_now)})
```

Comment:
Is there anything preventing us from turning this `headers` map into `Dict[str, Any]` instead of the `Dict[str, str]` that is is now. This would allow us to avoid having to do manual bool--> string conversion when using bools in headers.

Reply:
nit: can we add a constant that defines this header so it is easy to refer to in the future

Reply:
I think http headers has to be string since the protocol doesn't differentiate on their types. From API perspective, `r.Header.Get()` returns a string value, so it's always required to do an explicit type conversion here

Reply:
I see I'd rather we use `flag, err := strconv.ParseBool(flagStr)` in that case for parsing string to bool instead of explicitly checking a string match that has to match the upper or lower case exactly.